patches/net/minecraft/client/resources/SplashManager.java.patch

```diff
@@ -0,0 +1,44 @@
+--- a/net/minecraft/client/resources/SplashManager.java
++++ b/net/minecraft/client/resources/SplashManager.java
+@@ -34,12 +_,12 @@
+ 
+     protected List<String> prepare(ResourceManager p_118869_, ProfilerFiller p_118870_) {
+         try {
+-            List list;
++            List<String> list;
```

Comment:
The fact that the local is erased is _weird_ but maybe not worth a patch to fix? Add a cast in the other hunk.